keycloak/keycloak-admin-ui
# Kerberos and LDAP settings: Change hour and minute format to match old admin console UI

## Motivation
https://github.com/keycloak/keycloak-admin-ui/issues/322

## Brief Description
Fixes a few issues in the Cache section of both the Kerberos and LDAP settings pages:

- Fixes an issue in which the format of the minutes and hours drop-downs did not match the old admin console UI. Request was that we need to format them like the old console, with leading 0s when applicable. In the new console, the hour numbering now goes from 00-23 (as in the old console), instead of 1-24. In the new console, the minute numbering now goes from 00-59 (as in the old console), instead of 1-60.

- Fixes an issue in which the max lifespan fields were labeled and tagged as required (*). These fields accept blank values so removed the required tags.
 
- Fixed an issue with the days of the week sometimes not retrieving correctly from the server, and were resetting to Sunday (default). 

## Verification Steps
1. From the User Federation screen, click a Kerberos card to open settings.
2. In the Cache Settings section, select EVICT_WEEKLY or EVICT_DAILY from the Cache policy dropdown.
3. Verify that the hour and minute dropdowns list the correct values as noted above.
4. Verify that you can read/write these values correctly, preferably by checking them in the old console UI.
5. Select EVICT_WEEKLY and verify that you can read/write all days of the week correctly.
6. Select MAX_LIFESPAN from the Cache policy dropdown and verify that the Max lifespan field is no longer showing as required.
7. Repeat steps 1-6 for LDAP user federation settings. 

## Checklist:

## Additional Notes
None
